contracts/core/extensions/CoreIssuance.sol

```diff
@@ -283,6 +283,7 @@ contract CoreIssuance is
         uint256 _quantity
     )
         internal
+        whenOperational
```

Comment:
Since we're adding this at a lower level I think there's higher level `whenOperational` modifiers we can remove. Also does the same modifier need to be added for `issueInVaultInternal`?

Reply:
Hmm. true. Actually `issueInVaultInternal` uses issueInternal